Source/CBLInternal.h

```diff
@@ -160,4 +161,9 @@ NSString* CBLKeyPathForQueryRow(NSString* keyPath); // for testing
 - (instancetype) initWithDatabase: (CBLDatabase*)database
                            remote: (NSURL*)remote
                              pull: (BOOL)pull                       __attribute__((nonnull));
+@end
+
+
+@interface CBLCookieStorage ()
+- (void) reset;
```

Comment:
Yes. Actually I can make changes to the existing deleteAllCookies: method to set nil to the database info cookies key.